utils/openebs-cleanup.yml

```diff
@@ -1,5 +1,3 @@
-- hosts: localhost
```

Comment:
Why is this change required?

Reply:
This change is required because when we include file at that time host is not required
```TASK [include_tasks] ***********************************************************
fatal: [127.0.0.1]: FAILED! => {"reason": "conflicting action statements: hosts, tasks\n\nThe error appears to be in '/utils/openebs-label-cleanup.yaml': line 1, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- hosts: localhost\n  ^ here\n"}
```